spring-integration-jms/src/main/java/org/springframework/integration/jms/ChannelPublishingJmsMessageListener.java

```diff
@@ -518,6 +519,10 @@ public ErrorMessage buildErrorMessage(Throwable throwable) {
 			return super.buildErrorMessage(null, throwable);
 		}
 
+		MessagingTemplate getMessagingTemplate() {
```

Comment:
Because the outer class cannot access the delegate's protected `messagingTemplate field.